bibliopixel/drivers/driver_base.py

```diff
@@ -5,7 +5,15 @@
 
 
 class DriverBase(object):
-    """Base driver class to build other drivers from."""
+    """
+    Base driver class to build other drivers from.
+
+    :param int num: Number of total pixels held by this driver
+    :param int width: Width of matrix, for use with :py:class:`bibliopixel.layout.matrix.Matrix`
+    :param int height: Height of matrix, for use with :py:class:`bibliopixel.layout.matrix.Matrix`
+    :param str c_order: Color channel order
+    :param gamma: Gamma correction table. Preset tables available in :py:mod:`bibliopixel.util.colors.gamma`
```

Comment:
That sentence is ambiguous if you don't know what it means before you read it, because "preset" can also be a verb.  What about "Preset tables _are_ available"...?